application/common/models/User.php

```diff
@@ -54,7 +54,20 @@ public function beforeDelete()
                 return false;
             }
         }
-        
+
+        foreach ($this->methods as $method) {
+            if (! $mfa->delete()) {
```

Comment:
Looks like a typo here
```suggestion
            if (! $method->delete()) {
```